packages/model-viewer/src/features/ar.ts

```diff
@@ -106,6 +110,7 @@ export const ARMixin = <T extends Constructor<ModelViewerElementBase>>(
     protected[$arModes]: Set<ARMode> = new Set();
     protected[$arMode]: ARMode = ARMode.NONE;
     protected[$preload] = false;
+    private[$xrEnvironment] = false;
```

Comment:
No need for this; the `@property` above automatically creates a `this.xrEnvironment` member.

Reply:
Updated, thanks!